### PR TITLE
Added redmi note11s (fleur)

### PR DESCRIPTION
**Pull request by SysWhiteDev**

### CODE REVIEW

File: Xiaomi/RedmiNote11s/res/xml/power_profile.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        <value>0.1</value>
+    </array>
+    <item name="cpu.idle">0.1</item>
+    <item name="cpu.suspend">5</item>
+    <item name="cpu.active">2.55</item>. <item name="cpu.cluster_power.cluster0">2.11</item>
+    <item name="cpu.cluster_power.cluster1">2.22</item>
+    <array name="memory.bandwidths">
+        <value>22.7</value>
+    </array>
+    <item name="battery.capacity">1000</item>
```
Comment (phhusson) [inline Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: Yeah that's doubtful. You most likely picked those values from the wrong place.

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: sorry can i ask a question? how can i install it on device?

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: oh and that value was in stock rom of the device

Comment (phhusson) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: How did you take them from stock rom?

There is a whole set of overrides/overlays, where many different components can override others. So unless you actually asked the system the value of R.xml.power_profile, it's not "the value was in stock rom of the device"

You can install it with:
```
adb root
adb remount
adb push xxxx.apk /system/product/overlay/
```

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: mh ok ty, ill update it with the correct info as soon as i can

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: sorry to bother again, how to check if the overlay is being used by android?

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: bc it seems like on reboot my device removes the new files

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: oh and i like this project very much, how can i contribute with a small ammount of money?

Comment (phhusson) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: contribute with doing an overlay and spreading that project it's already good enough :-)

> bc it seems like on reboot my device removes the new files

Sounds like you have magisk installed. Uninstall it, at least the time to do those tests

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: i do not have magisk installed on this device

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: the device behaves like it ignores the overlay
btw is there to do a donation to the project?

Comment (phhusson) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: You can donate to me through https://github.com/sponsors/phhusson/

If the file disappear after reboot, give me the result of `adb shell mount`

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: like it gave me an hope to run custom rom on my device, so i really want to contribute in an "economic" way

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: the output of the command you gave me is

```
tmpfs on /dev type tmpfs (rw,seclabel,nosuid,relatime,size=2897376k,nr_inodes=724344,mode=755)
devpts on /dev/pts type devpts (rw,seclabel,relatime,mode=600,ptmxmode=000)
proc on /proc type proc (rw,relatime,gid=3009,hidepid=2)
sysfs on /sys type sysfs (rw,seclabel,relatime)
selinuxfs on /sys/fs/selinux type selinuxfs (rw,relatime)
tmpfs on /mnt type tmpfs (rw,seclabel,nosuid,nodev,noexec,relatime,size=2897376k,nr_inodes=724344,mode=755,gid=1000)
/dev/block/sdc20 on /metadata type ext4 (rw,seclabel,nosuid,nodev,noatime,discard,data=ordered)
/dev/block/dm-0 on / type ext4 (ro,seclabel,nodev,relatime)
/dev/block/dm-2 on /vendor type ext4 (ro,seclabel,relatime)
tmpfs on /apex type tmpfs (rw,seclabel,nosuid,nodev,noexec,relatime,size=2897376k,nr_inodes=724344,mode=755)
tmpfs on /linkerconfig type tmpfs (rw,seclabel,nosuid,nodev,noexec,relatime,size=2897376k,nr_inodes=724344,mode=755)
tmpfs on /mnt/installer type tmpfs (rw,seclabel,nosuid,nodev,noexec,relatime,size=2897376k,nr_inodes=724344,mode=755,gid=1000)
tmpfs on /mnt/androidwritable type tmpfs (rw,seclabel,nosuid,nodev,noexec,relatime,size=2897376k,nr_inodes=724344,mode=755,gid=1000)
none on /sys/fs/cgroup type cgroup2 (rw,nosuid,nodev,noexec,relatime)
none on /dev/cpuctl type cgroup (rw,nosuid,nodev,noexec,relatime,cpu)
none on /dev/cpuset type cgroup (rw,nosuid,nodev,noexec,relatime,cpuset,noprefix,release_agent=/sbin/cpuset_release_agent)
none on /dev/memcg type cgroup (rw,nosuid,nodev,noexec,relatime,memory)
none on /dev/stune type cgroup (rw,nosuid,nodev,noexec,relatime,schedtune)
tmpfs on /linkerconfig type tmpfs (rw,seclabel,nosuid,nodev,noexec,relatime,size=2897376k,nr_inodes=724344,mode=755)
/dev/block/dm-0 on /apex/com.android.adbd type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.adservices type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.apex.cts.shim type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.appsearch type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.art type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.btservices type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.cellbroadcast type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.conscrypt type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.extservices type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.i18n type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.ipsec type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.media type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.media.swcodec type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.mediaprovider type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.neuralnetworks type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.ondevicepersonalization type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.os.statsd type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.permission type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.resolv type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.runtime type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.scheduling type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.sdkext type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.tethering type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.tzdata type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.uwb type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.vndk.v33 type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.vndk.v28 type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.vndk.v29 type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.vndk.v30 type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.vndk.v31 type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.vndk.v32 type ext4 (ro,seclabel,relatime)
/dev/block/dm-0 on /apex/com.android.wifi type ext4 (ro,seclabel,relatime)
tracefs on /sys/kernel/tracing type tracefs (rw,seclabel,relatime,gid=3012)
none on /config type configfs (rw,nosuid,nodev,noexec,relatime)
binder on /dev/binderfs type binder (rw,relatime,max=1048576,stats=global)
none on /sys/fs/fuse/connections type fusectl (rw,relatime)
bpf on /sys/fs/bpf type bpf (rw,nosuid,nodev,noexec,relatime)
pstore on /sys/fs/pstore type pstore (rw,seclabel,nosuid,nodev,noexec,relatime)
/dev/block/sdc8 on /cache type ext4 (rw,seclabel,nosuid,nodev,noatime,discard,noauto_da_alloc,data=ordered)
/dev/block/sdc22 on /mnt/vendor/protect_f type ext4 (rw,seclabel,nosuid,nodev,noatime,nodelalloc,noauto_da_alloc,commit=1,data=ordered)
/dev/block/sdc23 on /mnt/vendor/protect_s type ext4 (rw,seclabel,nosuid,nodev,noatime,nodelalloc,noauto_da_alloc,commit=1,data=ordered)
/dev/block/sdc12 on /mnt/vendor/nvdata type ext4 (rw,seclabel,nosuid,nodev,noatime,discard,noauto_da_alloc,data=ordered)
/dev/block/sdc11 on /mnt/vendor/nvcfg type ext4 (rw,seclabel,nosuid,nodev,noatime,nodelalloc,noauto_da_alloc,commit=1,data=ordered)
/dev/block/sdc8 on /mnt/rescue type ext4 (rw,seclabel,nosuid,nodev,noatime,discard,noauto_da_alloc,data=ordered)
/dev/block/sdc21 on /mnt/vendor/persist type ext4 (rw,seclabel,nosuid,nodev,noatime,nodelalloc,noauto_da_alloc,commit=1,data=ordered)
tmpfs on /storage type tmpfs (rw,seclabel,nosuid,nodev,noexec,relatime,size=2897376k,nr_inodes=724344,mode=755,gid=1000)
adb on /dev/usb-ffs/adb type functionfs (rw,relatime)
mtp on /dev/usb-ffs/mtp type functionfs (rw,relatime)
ptp on /dev/usb-ffs/ptp type functionfs (rw,relatime)
none on /mnt/phh type tmpfs (rw,seclabel,nodev,relatime,size=2897376k,nr_inodes=724344,mode=755)
none on /system/usr/keylayout type tmpfs (rw,seclabel,nodev,relatime,size=2897376k,nr_inodes=724344,mode=755)
none on /vendor/lib64/libmtk-ril.so type tmpfs (rw,seclabel,nodev,relatime,size=2897376k,nr_inodes=724344,mode=755)
/dev/block/sdc21 on /persist type ext4 (rw,seclabel,nosuid,nodev,noatime,nodelalloc,noauto_da_alloc,commit=1,data=ordered)
/dev/block/dm-0 on /vendor/etc/usb_audio_policy_configuration.xml type ext4 (ro,seclabel,nodev,relatime)
/dev/block/dm-3 on /data type f2fs (rw,lazytime,seclabel,nosuid,nodev,noatime,background_gc=on,gc_merge,discard,no_heap,user_xattr,inline_xattr,acl,inline_data,inline_dentry,flush_merge,extent_cache,mode=adaptive,active_logs=6,reserve_root=54785,resuid=0,resgid=1065,inlinecrypt,alloc_mode=reuse,checkpoint_merge,fsync_mode=nobarrier)
/dev/block/dm-3 on /data/user/0 type f2fs (rw,lazytime,seclabel,nosuid,nodev,noatime,background_gc=on,gc_merge,discard,no_heap,user_xattr,inline_xattr,acl,inline_data,inline_dentry,flush_merge,extent_cache,mode=adaptive,active_logs=6,reserve_root=54785,resuid=0,resgid=1065,inlinecrypt,alloc_mode=reuse,checkpoint_merge,fsync_mode=nobarrier)
tmpfs on /data_mirror type tmpfs (rw,seclabel,nosuid,nodev,noexec,relatime,size=2897376k,nr_inodes=724344,mode=700,gid=1000)
/dev/block/dm-3 on /data_mirror/data_ce/null type f2fs (rw,lazytime,seclabel,nosuid,nodev,noatime,background_gc=on,gc_merge,discard,no_heap,user_xattr,inline_xattr,acl,inline_data,inline_dentry,flush_merge,extent_cache,mode=adaptive,active_logs=6,reserve_root=54785,resuid=0,resgid=1065,inlinecrypt,alloc_mode=reuse,checkpoint_merge,fsync_mode=nobarrier)
/dev/block/dm-3 on /data_mirror/data_ce/null/0 type f2fs (rw,lazytime,seclabel,nosuid,nodev,noatime,background_gc=on,gc_merge,discard,no_heap,user_xattr,inline_xattr,acl,inline_data,inline_dentry,flush_merge,extent_cache,mode=adaptive,active_logs=6,reserve_root=54785,resuid=0,resgid=1065,inlinecrypt,alloc_mode=reuse,checkpoint_merge,fsync_mode=nobarrier)
/dev/block/dm-3 on /data_mirror/data_de/null type f2fs (rw,lazytime,seclabel,nosuid,nodev,noatime,background_gc=on,gc_merge,discard,no_heap,user_xattr,inline_xattr,acl,inline_data,inline_dentry,flush_merge,extent_cache,mode=adaptive,active_logs=6,reserve_root=54785,resuid=0,resgid=1065,inlinecrypt,alloc_mode=reuse,checkpoint_merge,fsync_mode=nobarrier)
/dev/block/dm-3 on /data_mirror/cur_profiles type f2fs (rw,lazytime,seclabel,nosuid,nodev,noatime,background_gc=on,gc_merge,discard,no_heap,user_xattr,inline_xattr,acl,inline_data,inline_dentry,flush_merge,extent_cache,mode=adaptive,active_logs=6,reserve_root=54785,resuid=0,resgid=1065,inlinecrypt,alloc_mode=reuse,checkpoint_merge,fsync_mode=nobarrier)
/dev/block/dm-3 on /data_mirror/ref_profiles type f2fs (rw,lazytime,seclabel,nosuid,nodev,noatime,background_gc=on,gc_merge,discard,no_heap,user_xattr,inline_xattr,acl,inline_data,inline_dentry,flush_merge,extent_cache,mode=adaptive,active_logs=6,reserve_root=54785,resuid=0,resgid=1065,inlinecrypt,alloc_mode=reuse,checkpoint_merge,fsync_mode=nobarrier)
/data/media on /mnt/runtime/default/emulated type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,multiuser,mask=6,derive_gid,default_normal,unshared_obb)
/data/media on /mnt/runtime/read/emulated type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,multiuser,mask=23,derive_gid,default_normal,unshared_obb)
/data/media on /mnt/runtime/write/emulated type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,multiuser,mask=7,derive_gid,default_normal,unshared_obb)
/data/media on /mnt/runtime/full/emulated type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,multiuser,mask=7,derive_gid,default_normal,unshared_obb)
/dev/fuse on /mnt/user/0/emulated type fuse (rw,lazytime,nosuid,nodev,noexec,noatime,user_id=0,group_id=0,allow_other)
/dev/fuse on /mnt/installer/0/emulated type fuse (rw,lazytime,nosuid,nodev,noexec,noatime,user_id=0,group_id=0,allow_other)
/dev/fuse on /mnt/androidwritable/0/emulated type fuse (rw,lazytime,nosuid,nodev,noexec,noatime,user_id=0,group_id=0,allow_other)
/dev/fuse on /storage/emulated type fuse (rw,lazytime,nosuid,nodev,noexec,noatime,user_id=0,group_id=0,allow_other)
/data/media on /mnt/pass_through/0/emulated type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,multiuser,mask=7,derive_gid,default_normal,unshared_obb)
/data/media on /mnt/user/0/emulated/0/Android/data type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,multiuser,mask=6,derive_gid,default_normal,unshared_obb)
/data/media on /storage/emulated/0/Android/data type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,multiuser,mask=6,derive_gid,default_normal,unshared_obb)
/data/media on /mnt/androidwritable/0/emulated/0/Android/data type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,multiuser,mask=6,derive_gid,default_normal,unshared_obb)
/data/media on /mnt/installer/0/emulated/0/Android/data type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,multiuser,mask=6,derive_gid,default_normal,unshared_obb)
/data/media on /mnt/user/0/emulated/0/Android/obb type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,multiuser,mask=6,derive_gid,default_normal,unshared_obb)
/data/media on /storage/emulated/0/Android/obb type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,multiuser,mask=6,derive_gid,default_normal,unshared_obb)
/data/media on /mnt/androidwritable/0/emulated/0/Android/obb type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,multiuser,mask=6,derive_gid,default_normal,unshared_obb)
/data/media on /mnt/installer/0/emulated/0/Android/obb type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,multiuser,mask=7,derive_gid,default_normal,unshared_obb)
/dev/block/vold/public:179,1 on /mnt/media_rw/2216-1CEB type vfat (rw,dirsync,nosuid,nodev,noexec,noatime,gid=1023,fmask=0007,dmask=0007,allow_utime=0020,codepage=437,iocharset=iso8859-1,shortname=mixed,utf8,errors=remount-ro)
/mnt/media_rw/2216-1CEB on /mnt/runtime/default/2216-1CEB type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,mask=6)
/mnt/media_rw/2216-1CEB on /mnt/runtime/read/2216-1CEB type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,mask=18)
/mnt/media_rw/2216-1CEB on /mnt/runtime/write/2216-1CEB type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,mask=18)
/mnt/media_rw/2216-1CEB on /mnt/runtime/full/2216-1CEB type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,mask=7)
/dev/fuse on /mnt/user/0/2216-1CEB type fuse (rw,lazytime,nosuid,nodev,noexec,noatime,user_id=0,group_id=0,allow_other)
/dev/fuse on /mnt/installer/0/2216-1CEB type fuse (rw,lazytime,nosuid,nodev,noexec,noatime,user_id=0,group_id=0,allow_other)
/dev/fuse on /mnt/androidwritable/0/2216-1CEB type fuse (rw,lazytime,nosuid,nodev,noexec,noatime,user_id=0,group_id=0,allow_other)
/dev/fuse on /storage/2216-1CEB type fuse (rw,lazytime,nosuid,nodev,noexec,noatime,user_id=0,group_id=0,allow_other)
/mnt/media_rw/2216-1CEB on /mnt/pass_through/0/2216-1CEB type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,mask=7)

```

Comment (phhusson) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: Can you give the output on the console of the three adb commands I gave you?

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: image
here are the outputs

Comment (phhusson) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: Uh, the overlay that you create is not called framework-res.apk, it's treble-overlay-xiaomi-redminote11s

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: yes i know, i renamed it, should i have not done that?

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: i tried pushing the file with original name, but the phone keeps deleting it

Comment (SysWhiteDev) [reply Xiaomi/RedmiNote11s/res/xml/power_profile.xml:38]: question, i noticed that the status bar is wrongly sized, like it doesnt et the height of the camera, should the overlay fix this?

**Comment by SysWhiteDev** [comment]
i tried pushing the file with original name, but the phone keeps deleting it

**Comment by SysWhiteDev** [comment]
question, i noticed that the status bar is wrongly sized, like it doesnt et the height of the camera, should the overlay fix this?

**Comment by phhusson** [comment]
When is it deleted? Right after pushing, or after rebooting?
If the later, you can try to do `adb shell 'stop;start'` rather than rebooting to soft-reboot android without actually rebooting

**Comment by SysWhiteDev** [comment]
But then wouldn't it delete it if I need to reboot the phone?

**Comment by phhusson** [comment]
atm you're just testing the overlay though? once you tested it, it gets merged and goes into all GSIs

**Comment by SysWhiteDev** [comment]
Yea, but i would need to build one then? And I have no idea or the hardware to do that

**Comment by phhusson** [comment]
No? Just wait for whichever GSI you want to upgrade once your overlay has been merged?

**Comment by SysWhiteDev** [comment]
I mean, i rly just want it working rn, it would be nice if it got merged in all of them, but this would take time.

**Comment by SysWhiteDev** [comment]
Just tested this morning, seems that the device removes the .apk with adb shell stop; start too

**Comment by phhusson** [comment]
How do you check that it's removed?

19 déc. 2022 07:08:14 SysWhite ***@***.***>:

**Comment by SysWhiteDev** [comment]
I check by doing adb shell ls /system/product/overlay/ and check if the file I added is still there

**Comment by SysWhiteDev** [comment]
how should i check if the overlay its working?

**Comment by phhusson** [comment]
well you tried to fix stuff in the overlay, check that those things that you tried to fix are fixed?

If you're just asking whether it's loaded, you can run as root `lsof |grep /system/product/overlay/name-of-the-overlay-file.apk`. If there are matches, then it probably is being loaded

**Pull request closed without merging**